woocommerce/woocommerce-android
# Issue/6131 order creation product overlap

Fixes #6131 by changing the constraints on the product attribute view and divider view so long product names don't overlap.

To test:

* Start order creation
* Add a product whose name wraps more than one line
* Note that the name doesn't overlap the price
* Add a product with a single line name
* Note that it looks as expected

![product](https://user-images.githubusercontent.com/3903757/161254734-cc8aa223-c9da-4d6d-98da-efb166a72465.png)
